pycop/simulation.py

```diff
@@ -24,19 +26,22 @@ def simu_gaussian(n, m, corrMatrix):
         the simulated sample 
 
     """
-
+    if not all(isinstance(v, int) for v in [n, m]):
+        raise TypeError("The 'n' and 'm' arguments must both be integer types.")
+    if not isinstance(corr_matrix, np.array):
```

Comment:
Seems that for checking the type we need provide "np.ndarray" and not "np.array":
```suggestion
    if not isinstance(corr_matrix, np.ndarray):
```